apps/smp-server/Main.hs

```diff
@@ -173,8 +173,8 @@ fileExists path = do
   exists <- liftIO $ doesFileExist path
   unless exists . throwE $ "file " <> path <> " not found"
 
-deleteFileIfExists :: FilePath -> IO ()
-deleteFileIfExists path = doesFileExist keyPath >>= (`when` removeFile path)
+deleteIfExists :: FilePath -> IO ()
+deleteIfExists path = doesFileExist keyPath >>= (`when` removeFile path)
```

Comment:
should we maybe write:
`doesFileExist path >>= ...`

Reply:
fixed